django/db/models/base.py

```diff
@@ -58,6 +58,11 @@ def subclass_exception(name, bases, module, attached_to):
     })
 
 
+def _has_contribute_to_class(value):
+    # We should call the contribute_to_class method only if it's bound.
```

Comment:
I would remove _"We should"_.

Reply:
And move _"only"_ to the start - the sentence flows better.